Sources/swift-container-plugin/Documentation.docc/Adding-the-plugin-to-your-project.md

```diff
@@ -0,0 +1,34 @@
+# Add the plugin to your project
+
+Make Swift Container Plugin available in your project
+
+## Overview
+
+Swift Container Plugin is distributed as a Swift Package Manager package.    To make it available, you must add it as a dependency of your project.
+
+### Install the plugin using the `swift package` CLI
+
+Recent versions of `swift package` suupport the `add-dependency` command:
```

Comment:
I just found out that the add-dependency command _doesn't_ check to see if the dependency is already added... so adding it twice caused an issue when I ran `swift package plugin --list` to see if it was available. 

I used the HummingbirdVaporExample in the examples/ directory, which already had the plugin included. Might be worth adding "If your project doesn't already have the plugin included..." as a prior statement to work around this unfortunate side effect.

Reply:
Yes, I've seen that before.   I couldn't find an existing issue about it so I've created https://github.com/swiftlang/swift-package-manager/issues/8519

Reply:
How about this?
```suggestion
> Adding the same dependency to your project more than once will prevent it from building.
> If this happens, delete the duplicate dependency lines from `Package.swift` and rebuild.

Recent versions of `swift package` suupport the `add-dependency` command:
```

Reply:
looks good!